students/swietczak_monika/lesson_03_functions/calculator.py

```diff
@@ -0,0 +1,61 @@
+def calculate(option, a, b):
+    """
+    Performs calculations according to selected option
+    :param option: string, one of selected options
+    :param a: integer
+    :param b: integer
+    :return: number
+    """
+    if option == "a":
+        return a + b
+    if option == "s":
```

Comment:
For code readability it's better to use elif here and handle default case with else when incorrect option was provided